std/zip.d

```diff
@@ -1700,6 +1768,17 @@ the quick brown fox jumps over the lazy dog\r
     assert(za.directory.length == 0);
 }
 
+@system unittest
```

Comment:
whats the reason this can't be `@safe`

Reply:
The `cast` to `void`.

Reply:
Hmm, could you have that as part of the initialiser of `file`?

Reply:
Oh I see all the other tests a re like that too, never mind.